packages/assets-controllers/src/NftController.ts

```diff
@@ -1511,32 +1549,34 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
     tokenId: string,
     {
       networkClientId,
-      userAddress = this.config.selectedAddress,
+      userAddress = this.#config.selectedAddress,
     }: { networkClientId?: NetworkClientId; userAddress?: string } = {
-      userAddress: this.config.selectedAddress,
+      userAddress: this.#config.selectedAddress,
     },
   ) {
-    const chainId = this.getCorrectChainId({ networkClientId });
-    address = toChecksumHexAddress(address);
-    this.removeAndIgnoreIndividualNft(address, tokenId, {
+    const chainId = this.#getCorrectChainId({ networkClientId });
+    const checksumHexAddress = toChecksumHexAddress(address);
+    this.#removeAndIgnoreIndividualNft(checksumHexAddress, tokenId, {
       chainId,
       userAddress,
     });
     const { allNfts } = this.state;
     const nfts = allNfts[userAddress]?.[chainId] || [];
     const remainingNft = nfts.find(
-      (nft) => nft.address.toLowerCase() === address.toLowerCase(),
+      (nft) => nft.address.toLowerCase() === checksumHexAddress.toLowerCase(),
     );
     if (!remainingNft) {
-      this.removeNftContract(address, { chainId, userAddress });
+      this.#removeNftContract(checksumHexAddress, { chainId, userAddress });
     }
   }
 
   /**
    * Removes all NFTs from the ignored list.
    */
   clearIgnoredNfts() {
-    this.update({ ignoredNfts: [] });
+    this.update((state) => {
+      state.ignoredNfts = [];
+    });
   }
```

Comment:
Is there a reason we're changing this? I think the `update` method should be able to handle the original object argument.

Reply:
no this pattern doesn't work with v2 controllers as this.update takes a callback func as param

Reply:
Oh you're right! My bad I got confused.